### ISSUE TITLE
Process 2006-10 data

### ISSUE THREAD

**Issue by lhendey** [NeighborhoodInfoDC/ACS#86]
Instructions are here. https://urbanorg.app.box.com/notes/51315236817

Complete after issue #87

**Comment by lhendey** [comment]
@Woliver01 - thanks for all your work last week. it looks like this issue is still outstanding?

**Comment by Woliver01** [comment]
@lhendey I believe I did most of these steps under issues #91 and #87. It looks like I still need to commit and sync the changes for this issue so I'll do that now.

**Comment by lhendey** [comment]
@Woliver01 - there are no 2006_10 files on L:\Libraries\ACS\Data  (raw or processed).  this is the step that needs to be done before we can do any summary data.

**Comment by Woliver01** [comment]
I'll have those up shortly. @lhendey

**Comment by lhendey** [comment]
@Woliver01 thanks - the programs are also not up there so they probably need to be committed and new pull request done.
